docs/quickstarts/javascript/react/conditional-ui-rendering.mdx

```diff
@@ -220,6 +203,17 @@ if (asertoError) {
 }
 ```
 
+Next, we’ll add a to call `getDisplayState`:
+
+```javascript
+const displayStateMap =
```

Comment:
I would call this "displayState" not "displayStateMap" - the displayStateMap is a map of maps